cli/internal/nodeprovider/local/createNodeIfNotExists.go

```diff
@@ -76,9 +69,38 @@ func (np nodeProvider) CreateNodeIfNotExists(ctx context.Context) (node.Node, er
 		return nil, err
 	}
 
-	err = apiClientNode.Liveness(ctx)
-	nodeLogBytes, _ := os.ReadFile(nodeLogFilePath)
-	fmt.Println(string(nodeLogBytes))
+	ctx, cancel := context.WithCancel(ctx)
+
+	go func() {
+		for {
+			select {
+			case <-ctx.Done():
+				return
+			default:
+				if err := apiClientNode.Liveness(ctx); err == nil {
+					cancel()
+				}
+				time.Sleep(time.Second)
```

Comment:
@zachpuck replaced with this loop. This was the only place we "relied" on that retry behavior from that 3p lib, so I thought reasonable to simplify. It also helps startup faster because the 3p lib does exponential backoff whereas here we just do linear.